<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cppcheck::GenericConfig</class>
 <widget class="QWidget" name="cppcheck::GenericConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>349</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel2">
     <property name="text">
      <string>Cppcheck executable:</string>
     </property>
     <property name="buddy">
      <cstring>cppcheckExecutable</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KUrlRequester" name="cppcheckExecutable"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Extra &amp;parameters:</string>
     </property>
     <property name="buddy">
      <cstring>cppcheckParameters</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KLineEdit" name="cppcheckParameters"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="AdditionalChecksGroupBox">
     <property name="toolTip">
      <string>Enable additional checks</string>
     </property>
     <property name="title">
      <string>Additional Checks</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="styleCheckBox">
        <property name="toolTip">
         <string>Check coding style</string>
        </property>
        <property name="text">
         <string>style</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="performanceCheckBox">
        <property name="toolTip">
         <string>Enable performance messages</string>
        </property>
        <property name="text">
         <string>performance</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="portabilityCheckBox">
        <property name="toolTip">
         <string>Enable portability messages</string>
        </property>
        <property name="text">
         <string>portability</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="informationCheckBox">
        <property name="toolTip">
         <string>Enable information messages</string>
        </property>
        <property name="text">
         <string>information</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="unusedFunctionCheckBox">
        <property name="toolTip">
         <string>Check for unused functions</string>
        </property>
        <property name="text">
         <string>unused function</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="missingIncludeCheckBox">
        <property name="toolTip">
         <string>Warn if there are missing includes</string>
        </property>
        <property name="text">
         <string>missingInclude</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>88</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="SavePushButton">
       <property name="text">
        <string>&amp;Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
